web/src/pages/Main.tsx

```diff
@@ -1,14 +1,65 @@
 import { Header } from '../components/common/Header'
-import { useContext } from 'react'
-import { AuthContext } from 'react-oauth2-code-pkce'
+import { useEffect, useState } from 'react'
+import { MoleTable } from '../components/feature/MoleTable'
+import MercuryAPI from '../api/MercuryAPI'
+import { AxiosResponse } from 'axios'
+
+// TODO: Get type from generated API
+export type TComponentResponse = {
+  [key: string]: { [key: string]: string }
+}
 
 export const MainPage = (): JSX.Element => {
-  const { token } = useContext(AuthContext)
+  const [components, setComponents] = useState<TComponentResponse>()
+  const [loading, setLoading] = useState<boolean>(false)
```

Comment:
Can we use naming conventions for booleans? `isLoading`?

Reply:
done

Reply:
I think it got lost in the rebase?

Reply:
Yes. fixed :)